doc/api/assert.md

```diff
@@ -33,13 +33,16 @@ added: v0.1.21
 -->
 
 Tests for deep equality between the `actual` and `expected` parameters.
-Primitive values are compared with the equal comparison operator ( `==` ).
+Primitive values are compared with the [Abstract Equality Comparison][]
+( `==` ).
 
-Only enumerable "own" properties are considered. The `deepEqual()`
-implementation does not test object prototypes, attached symbols, or
-non-enumerable properties. This can lead to some potentially surprising
-results. For example, the following example does not throw an `AssertionError`
-because the properties on the [`Error`][] object are non-enumerable:
+Only [enumerable "own" properties][] are considered. The `deepEqual()`
+implementation does not test the [`[[Prototype]]`][prototype-spec] of
+objects, attached symbols, or non-enumerable properties. (For those checks,
```

Comment:
Oh yes, I was thinking parens look a bit awkward too, the hyphen does the trick!
I'll use `assert.deepStrictEqual()` because it has a link already. Thanks!